frontend/src/app/modules/fields/edit/field-types/float-edit-field.component.ts

```diff
@@ -27,22 +27,96 @@
 
 import {Component} from "@angular/core";
 import {EditFieldComponent} from "core-app/modules/fields/edit/edit-field.component";
+import {keyCodes} from "core-app/modules/common/keyCodes.enum";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/cb362a4f9bde7e8b78dc5ea0b921de24830d93fc/frontend/src/app/modules/fields/edit/field-types/float-edit-field.component.ts#L31"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>